ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -808,13 +801,10 @@ namespace OrdinalAxis {
             ordinalIndex = getIndexInArray(ordinalPositions, val);
             // final return value is based on ordinalIndex
         } else {
-
-            if (!extendedOrdinalPositions) {
-                extendedOrdinalPositions =
-                    ordinal.getExtendedPositions &&
-                    ordinal.getExtendedPositions();
-                ordinal.extendedOrdinalPositions = extendedOrdinalPositions;
-            }
+            extendedOrdinalPositions =
+                ordinal.getExtendedPositions &&
+                ordinal.getExtendedPositions();
+            ordinal.extendedOrdinalPositions = extendedOrdinalPositions;
```

Comment:
Unnecessary line + I think you can remove from definitions `Ordinal.extendedOrdinalPositions` too